use-cases/dlstreamer/docker-compose_replicas_gst.yml

```diff
@@ -0,0 +1,127 @@
+        rtsp://localhost:8554/camera_0
+        "
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ../../performance-tools/sample-media:/home/pipeline-server/sample-media
+  camera-simulator1:
+    build:
+      context: .
+    image: openvino/ubuntu20_data_runtime:2021.4.2
+    container_name: camera-simulator1
+    network_mode: "host"
+    entrypoint: ffmpeg
+    command: "
+        -nostdin
+        -re -stream_loop -1
+        -i /home/pipeline-server/sample-media/vehicle-bike-1920-15-bench.mp4
+        -c copy
+        -f rtsp
+        -rtsp_transport
+        tcp
+        rtsp://localhost:8554/camera_1
+        "
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ../../performance-tools/sample-media:/home/pipeline-server/sample-media
+  camera-simulator2:
+    build:
+      context: .
+    image: openvino/ubuntu20_data_runtime:2021.4.2
+    container_name: camera-simulator2
+    network_mode: "host"
+    entrypoint: ffmpeg
+    command: "
+        -nostdin
+        -re -stream_loop -1
+        -i /home/pipeline-server/sample-media/video_of_people_walking_855564-1920-15-bench.mp4
+        -c copy
+        -f rtsp
+        -rtsp_transport
+        tcp
+        rtsp://localhost:8554/camera_2
+        "
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ../../performance-tools/sample-media:/home/pipeline-server/sample-media
+  # OvmsServer:
+  #   image: openvino/model_server:2023.1-gpu
+  #   container_name: ovms-server0
+  #   restart: always
+  #   ports:
+  #     - '9001:9001'
+  #     - '9002:9002'
+  #   command:
+  #     - '--config_path'
+  #     - '/models/config.json'
+  #     - '--port'
+  #     - '9001'
+  #     - '--rest_port'
+  #     - '9002'
+  #     - '--log_level'
+  #     - 'INFO'
+  #   volumes:
+  #     - ./configs/opencv-ovms/models/2022:/models
+  #   environment:
+  #     - DEVICE=CPU update_config:dev
+  #     - PLATFORM="core"
+  #     - server_cl_cache_dir=/home/pipeline-server/.cl-cache
+  OvmsClientGst:
+    image: dlstreamer:dev
+    deploy:
+      mode: replicated
+      replicas: ${PIPELINE_COUNT}
+    network_mode: "host"
+    entrypoint: /script/entrypoint.sh --pipeline_script_choice ${PIPELINE_SCRIPT}
+    env_file:
+      - ./res/gst.env
+      - ./res/yolov5-cpu.env
+    environment:
+      - CONTAINER_NAME="gst0"
+      - GRPC_PORT=9001
+      - INPUTSRC=rtsp://localhost:8554/camera_1
+      - RENDER_MODE=0 #RENDER_MODE=1 will work only after running xhost +local:docker
+      - DISPLAY=$DISPLAY
+      - PIPELINE_SCRIPT=${PIPELINE_SCRIPT}
+    volumes:
+      - ../../results:/tmp/results
+      - ~/.Xauthority:/home/dlstreamer/.Xauthority
+      - /tmp/.X11-unix
+      - ~/.cl-cache:/home/pipeline-server/.cl-cache
+      - ../../performance-tools/sample-media/:/home/pipeline-server/vids
+      - ./res/:/home/pipeline-server/envs
+      - ./extensions:/home/pipeline-server/extensions
+      - ./framework-pipelines:/home/pipeline-server/framework-pipelines
```

Comment:
extensions and framework-pipelines are copied in the Dockerfile. The volumes can be removed.